src/views/germplasm/BreedingMethods.vue

```diff
@@ -576,7 +581,12 @@ export default class BreedingMethods extends ProgramsBase {
       this.getBreedingMethods();
       this.$emit('show-success-notification', 'Breeding method updated successfully');
     } catch (e) {
-      this.$emit('show-error-notification', 'Error while trying to update breeding method');
+      if (e.response.status == 500) {
+        this.$emit('show-error-notification', 'Error while trying to create breeding method');
+      }
+      else{
+        this.$emit('show-error-notification', e.response.data);
+      }
```

Comment:
Whatever you decide to do in the *create* case, do in the *update* case as well.
```suggestion
      this.$emit('show-error-notification', e.response.data);
```

Reply:
Done.  Thanks.